src/Components/Publishing/Series/ArticleCard.tsx

```diff
@@ -17,38 +17,70 @@ interface Props {
 export class ArticleCard extends Component<Props, null> {
   public static defaultProps: Partial<Props>
 
+  isUnpublishedMedia () {
+    const { media } = this.props.article
+    return media && !media.published
+  }
+
   renderMediaDate () {
     const { article } = this.props
     const mediaDate = getMediaDate(article)
   
-    if (dateIsFuture(article.media.release_date)) {
+    if (this.isUnpublishedMedia()) {
       return (
-          <ComingSoon>
-            <span>Coming Soon</span>
+          <MediaDate>
+            <span>Available </span>
             <Date layout='condensed' date={mediaDate} />
-          </ComingSoon>
+          </MediaDate>
       )
     } else {
       return <Date layout='condensed' date={mediaDate} />
     }
   }
 
+  renderMediaCoverInfo () {
+    const { article, color } = this.props
+
+    if (this.isUnpublishedMedia()) {
+      return <MediaComingSoon>Coming Soon</MediaComingSoon>
+    } else {
+      return (
+        <MediaPlay>
+          <IconPlayCaret color={color} />
+          {article.media.duration}
+        </MediaPlay>
+      )
+    }
+  }
+
+  openLink = (e) => {
+    e.preventDefault()
+
+    if (!this.isUnpublishedMedia()) {
+      window.open(e.currentTarget.attributes.href.value)
+    }
+  }
```

Comment:
Ooh, is this because empty `href=""` will jump to the top of the page or something?

Reply:
Yes exactly